fluent-bundle/src/resolver/inline_expression.rs

```diff
@@ -160,16 +160,18 @@ impl<'p> ResolveValue for ast::InlineExpression<&'p str> {
             Self::StringLiteral { value } => unescape_unicode_to_string(value).into(),
             Self::NumberLiteral { value } => FluentValue::try_number(*value),
             Self::VariableReference { id } => {
-                let args = scope.local_args.as_ref().or(scope.args);
```

Comment:
Thought: These changes seem reasonable reading through it.

Question: Do you know if this code is explicitly covered by tests? I'd like to doubly make sure there is no behavior change here.

Reply:
I've done some experimentation right now to see if this is properly covered (deleting and swapping branches in the if condition) and there are tests that cover this in "fixtures".

I vaguely remember that I have look at related tests when touching this code to have a better grasp, if I'm not wrong the "Parameterized term attributes" test suite in `tests/fixtures/macros.yaml` are related to this, for example swapping if branches break this test: https://github.com/projectfluent/fluent-rs/blob/c91d9f48b5a8734bd9183145320f5108526b2d1e/fluent-bundle/tests/fixtures/macros.yaml#L358-L364

Do you think it's ok like this?

Reply:
Good enough for merging I think!